Readme.md

```diff
@@ -71,8 +71,13 @@ authenticate users. This file is uploaded to AWS secrets manager using `make set
   is only used when fusillade is first deployed to create the first users. Afterwards this variable has no effect. If
   more admins are required assign a user the admin role.
 - Environment variables can be set in `environment.local` for convenience.
-- **Optionally** modify the [default policies and roles](../blob/master/policies) to suite your needs prior to 
-  deployment. 
+- **Optionally** Before deploying fusillade you can modify the [default policies and roles](../blob/master/policies) 
+ to suite your 
+ needs. The `default_admin_role.json` is policy attached to the fusillade_admin role created during deployment. The 
+ `default_group_policy.json` is assigned to all new group when they are created. The `default_user_role.json` is the 
+ role
```

Comment:
Nit: the newlines might be a bit wonky here?

Reply:
fixed